cmd/bootkube/start.go

```diff
@@ -40,6 +43,12 @@ func runCmdStart(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("Invalid etcd etcdServer %q: %v", startOpts.etcdServer, err)
 	}
 
+	if startOpts.selfHostedEtcd {
```

Comment:
I'm fine leaving this as-is for now to keep this self-contained/simple -- but this should likely move into `bootkube.Run()` eventually.